tasks/update-infra-deployments.yaml

```diff
@@ -23,7 +26,7 @@ spec:
           # 'appstudio-staging-ci' - private key for appstudio-staging-ci Github app
   steps:
     - name: update-infra-deployments
-      image: registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8@sha256:af7dd5b3b1598a980f17d5f5d3d8a4b11ab4f5184677f7f17ad302baa36bd3c1
+      image: appstudio-utils
```

Comment:
Using this image so I can leverage the `yq` command in the `SCRIPT`.

Reply:
Does appstudio-utils have enough python installed?